src/search/utils/QueryStringGenerator.ts

```diff
@@ -31,11 +31,15 @@ const createTermString = (
   term: string | undefined,
   itemType: string,
   id: string | undefined,
-  termSuffix: boolean | undefined
+  termSuffix: boolean | undefined,
+  stringValue: string | undefined
 ) => {
   if (term === undefined) {
     throw new Error('term is undefined');
   }
+  if (term === 'xref' && stringValue === '*') {
```

Comment:
I don't know if whitespace is trimmed from `stringValue` anywhere but if not it probably should.

Reply:
Totally agree. I've just a commit that does all the trimming at the point of query string generation as the query builder form elements are controlled.